pkg/event/target/nsq_test.go

```diff
@@ -0,0 +1,157 @@
+/*
+ * (C) 2018 Matthias Schneider
```

Comment:
The copyright should be `Minio Cloud Storage 2018` we do not accept personal copyright headers

Reply:
This is primarily to do with defending the copyright on this work, if you in your personal capacity are willing to defend the copyright for now and in future then it is fine. But if we are going to defend it - it should be copyrighted to `Minio Cloud Storage`